content/docs/performance-and-scalability/scalability.md

```diff
@@ -18,12 +18,14 @@ weight: 60
 
 Current recommendations (when using all Istio features):
 
-* 1 vCPU per peak thousand requests per second for the sidecar(s)
+* 1 vCPU per peak thousand requests per second for the sidecar(s) with access logging and 0.5 without.
```

Comment:
For the 'access logging' case can you add that this is mostly a result of fluentd capturing and uploading the access logs. Porbably also useful to clarify which scenario is on by default.

Reply:
changed, ptal